app/views/layouts/govuk_publishing_components/preview.html.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
@fofr Mainly the stuff inside the body, so preview has the hide-header-footer class (and doesn't include any header content). Essentially, application.html includes more stuff in the body